repose-aggregator/src/docs/asciidoc/filters/herp.adoc

```diff
@@ -1,3 +1,213 @@
-= Highly Efficient Record Processor Filter
+= Highly Efficient Record Processor (HERP) Filter
 
 include::../_includes/in-progress.adoc[]
+
+The Highly Efficient Record Processor (HERP) filter logs an event for each API request.
+These logs provide information regarding the processing of each request.
+The logs can be inspected to view the usage of a service and to perform auditing.
+Events are recorded even if the API request is rejected by Repose before reaching the origin service.
+
+== General filter information
+* *Filter name:* HERP
+* *Filter configuration:* highly-efficient-record-processor.cfg.xml
+* *Released:* v7.0.0.1
```

Comment:
I'm curious what the reason is for including the release version. That's probably a discussion that's easier to have in person. I'll ask at lunch, probably. 😛